package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "dependencies": {}
+}
```

Comment:
What is `package.json` this file doing here? Committed by accident or is it really needed for something?